packages/ui-components/style/base.css

```diff
@@ -19,41 +19,6 @@
 @import './switch.css';
 @import './toolbar.css';
 
-/* Override Blueprint's _reset.scss styles */
-html {
-  box-sizing: unset;
-}
-
-*,
-*::before,
-*::after {
-  box-sizing: unset;
-}
-
-body {
-  color: var(--jp-ui-font-color1);
-  font-size: var(--jp-ui-font-size1);
```

Comment:
These two rules look like possibly important rules to keep, unless they are set elsewhere? Other than that, I'm so glad to see these go!

Reply:
I have added them back. Thx @jasongrout